gui

```diff
@@ -172,6 +172,8 @@ runonce <<"EOF"
   refresh_all_pkgapp_status
 EOF
 
+runonce <<"EOF"
+  [ -a "$HOME/.local/share/applications/pi-apps-settings.desktop" ] && sed -i "s/logo.png/settings.png/g" "$HOME/.local/share/applications/pi-apps-settings.desktop"
```

Comment:
I think an `EOF` is needed at the end as well.

Reply:
Fixed. Thanks for noticing.